IoTuring/Entity/Deployments/Fanspeed/Fanspeed.py

```diff
@@ -0,0 +1,261 @@
+import psutil
+
+from IoTuring.Entity.Entity import Entity
+from IoTuring.Entity.EntityData import EntitySensor
+from IoTuring.Entity.ValueFormat import ValueFormatterOptions
+from IoTuring.MyApp.SystemConsts import OperatingSystemDetection as OsD
+from IoTuring.Configurator.MenuPreset import MenuPreset
+
+
+supports_win_fanspeed = False
+supports_linux_fanspeed = True
+supports_macos_fanspeed = False
+
+KEY_FANSPEED = "fanspeed"
+KEY_FANLABEL = "fanlabel"
+
+FALLBACK_PACKAGE_LABEL = "controller"
+FALLBACK_SENSOR_LABEL = "fan"
+
+CONFIG_KEY_CONTROLLER = "controller"
+CONFIG_KEY_THRESHOLD = "threshold"
+
+FANSPEED_DECIMALS = 0
+
+
+class Fanspeed(Entity):
+    """Entity to read fanspeed"""
+
+    NAME = "Fanspeed"
+
+    def Initialize(self) -> None:
+        """Initialize the Class, setup Formatter, determin specificInitialize and specificUpdate depending on OS"""
+        self.fanspeedFormatOptions = ValueFormatterOptions(
+            value_type=ValueFormatterOptions.TYPE_ROTATION, decimals=FANSPEED_DECIMALS
+        )
+
+        self.specificInitialize = None
+        self.specificUpdate = None
+
+        if OsD.IsLinux():
+            self.specificInitialize = self.InitLinux
+            self.specificUpdate = self.UpdateLinux
+        self.specificInitialize()
+
+    def InitLinux(self) -> None:
+        """OS dependant Init for Linux"""
+        self.configuredPackages: list[str] = []
+        self.registeredPackages: list[psutilFanspeedPackage] = []
+        self.configuredThreshold: int 
+        sensors = psutil.sensors_fans()
+        self.Log(self.LOG_DEBUG, f"fancontrollers found:{sensors}")
+        # load all controllers from config
+        self.config = self.GetConfigurations()
+        self.configuredThreshold = self.config[CONFIG_KEY_THRESHOLD]
+        if self.configuredThreshold is None:
+            self.configuredThreshold = 200
```

Comment:
Since the configuration question is set to non mandatory, you can set a default there so that you don't have to check if the value is none or something else; hae you tried it ?